nasConvert/ProtocolConfigurationOptions.go

```diff
@@ -12,7 +12,7 @@ import (
 
 type ProtocolOrContainerUnit struct {
 	ProtocolOrContainerID uint16
-	LengthofContents      uint8
+	LengthOfContents      uint8
```

Comment:
This just makes it consistent with the naming rules in other places (e.g. `UEPresenceInAreaOfInterestList`).